Schema_refinery/schema_refinery.py

```diff
@@ -0,0 +1,125 @@
+"""
+Purpose
+-------
+This file call all other modules of the packages, containing several arguments
+as input depending on the desired module.
+
+Code documentation
+------------------
+"""
+import sys
+import argparse
+
+try:
+    from DownloadAssemblies import download_assemblies
+except ModuleNotFoundError:
+    from Schema_refinery.DownloadAssemblies import download_assemblies
+
+def download_module():
+    """
+    Function that contains the required arguments for the download_module of the
+    schema refinery package, this function exports the arguments to
+    download_asseblies.py
+    """
+    parser = argparse.ArgumentParser(description=__doc__,
+                                     formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    parser.add_argument('Download_module', nargs='+',
+                        help='')
+
+    #Common arguments between databases
+    parser.add_argument('-db', '--database', type=str,
+                        required=True, dest='database',
+                        nargs='+',
+                        choices = ['NCBI','ENA661K'],
+                        help='Databases from which assemblies will be downloaded. ')
+
+    parser.add_argument('-s', '--species', type=str,
+                        required=True, dest='species',
+                        help='Desired species to fetch')
+
+    parser.add_argument('-o', '--output-directory', type=str,
+                        required=True, dest='output_directory',
+                        help='Output Path')
```

Comment:
Suggest changing to `help='Path to the output directory.'`. Is it problematic if the output directory exists?